test/services/teams/helper/helper.hook.js

```diff
@@ -0,0 +1,133 @@
+const { BadRequest } = require('feathers-errors');
+const service = require('feathers-mongoose');
+
+const _TYPE = ['before', 'after'];
+const _METHOD = ['get', 'update', 'patch', 'create', 'find', 'remove'];
+
+const _DefaultServiceOptions = {
+	Model: undefined,		//maybe must added fake?
+	paginate: {
+		default: 10,
+		max: 100
+	},
+	lean: true
+};
+
+const _DefaultHeaderParams = {
+	authorization:'<jwtToken>', 
+	host: process.env.HOST || 'localhost:3030', 
+	accept: 'application/json', 
+	'content-type':'application/json', 
+	connection: 'close' 
+};
+
+const isType = (type) => {
+	if (_TYPE.includes(type))
+		return type;
+	throw BadRequest('wrong hook type');
+};
+
+const isMethod = (method) => {
+	if (_METHOD.includes(method))
+		return method;
+	throw BadRequest('wrong hook method');
+};
+
+const isObject = (obj) => {
+	if (typeof obj === 'object')
+		return obj;
+	throw BadRequest('Must be an object.');
+};
+
+const getService=(options={})=>{
+	const opt = Object.assign({},_DefaultServiceOptions,options);
+	if(opt.Model===undefined || opt.paginate===undefined){
+		opt._TEST_INFO_='service can not create and is faked';
+		return opt;
+	}
+	return service(opt);
+};
+
+const patchParams = (params,account={}) => {
+	params.provider = params.provider || 'rest';
+	params.query = params.query || {};
+	params.authenticated = params.authenticated || true;  
+	params.headers = params.headers || {};
+	params.payload = params.payload || {accountId:account._id||'<accountId>'};
+	params.headers = params.headers || _DefaultHeaderParams;
+	params.account = params.account || account;
+	return params;
+};
+
+/*** feathers is hook test ***/
+var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ?
+	function (obj) {
+		return typeof obj;
+	} :
+	function (obj) {
+		return obj && typeof Symbol === "function" && obj.constructor === Symbol && obj !== Symbol.prototype ? "symbol" : typeof obj;
+	};
+
+function isHookObject(hookObject) {
+	return (typeof hookObject === 'undefined' ? 'undefined' : _typeof(hookObject)) === 'object' && typeof hookObject.method === 'string' && typeof hookObject.type === 'string';
+}
+
+
+
+//todo: add service (?)
+//todo: add missing params : query:Object , authenticated:true  ...etc..
```

Comment:
Will these todos be addressed in this PR or later? If later, I would suggest opening a ticket instead of  a comment (which we will surely forget)